superset/models/sql_lab.py

```diff
@@ -279,6 +284,27 @@ def default_endpoint(self) -> str:
     def get_extra_cache_keys(query_obj: Dict[str, Any]) -> List[str]:
         return []
 
+    @property
+    def tracking_url(self) -> Optional[str]:
```

Comment:
Could this just be a method called `get_tracking_url` and then you wouldn't need to rename the column virtually or have a getter/setter.

Reply:
@dpgaspar @betodealmeida do you have any recommendation on what's the best practice for such use case?